src/config.ts

```diff
@@ -149,11 +151,22 @@ export const LIQUIDITY_POOL_REGISTRY_ADDRESS: string | undefined = _.isEmpty(
           EnvVarType.ETHAddressHex,
       );
 
-export const RFQT_API_KEY_WHITELIST: string[] =
-    process.env.RFQT_API_KEY_WHITELIST === undefined ? [] : process.env.RFQT_API_KEY_WHITELIST.split(',');
+export const RFQT_API_KEY_WHITELIST: string[] = _.isEmpty(process.env.RFQT_API_KEY_WHITELIST)
+    ? []
+    : assertEnvVarType('RFQT_API_KEY_WHITELIST', process.env.RFQT_API_KEY_WHITELIST, EnvVarType.StringList);
+
+export const RFQT_MAKER_ASSET_OFFERINGS: RfqtMakerAssetOfferings = _.isEmpty(process.env.RFQT_MAKER_ASSET_OFFERINGS)
```

Comment:
Just an opinion, not a blocker.

But this feels backwards to me, we'd need to redeploy and have constant communication to list and delist certain pairs as RFQT providers come online.

I'd prefer to see the MM advertise what they support and we periodically fetch that inside AS. Like the SRA asset_pairs endpoint.

Reply:
We will def want to do something like that in a future revision 👍 

For now, MMers are not updating their pairs too often, so this is 'good enough' for a first approach.  The most important thing for now is that we don't waste precious resources making a query for a pair we know the MMer doesn't have.